modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,217 @@
+<3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
+<4> Specify the host name for the cluster.
+<5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
+<6> Specify the desired IP for the host. 
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f ibi-image-cluster-install.yaml.yaml
+----
+
+. Create the `ClusterDeployment` resource:
+
+.. Create a YAML file that defines the `ClusterDeployment` resource:
++
+.Example `ibi-cluster-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: ibi-cluster-deployment <1>
+  namespace: ibi-test
+spec:
+  baseDomain: example.com <2>
+  clusterInstallRef:
+    group: extensions.hive.openshift.io
+    kind: ImageClusterInstall 
+    name: ibi-image-install <3>
+    version: v1alpha1 <4>
+  clusterMetadata:
+    adminKubeconfigSecretRef:
+      name: ibi-test-admin-kubeconfig <5>
+    adminPasswordSecretRef:
+      name: ibi-test-admin-kubeconfig <6>
+    clusterID: da237966-4685-447d-b704-2d4e52a4345c <7>
+    infraID: ibiotest-t4k6t <8>
+  clusterName: ibi-cluster <9>
+  platform:
+    none: {} <10>
+  pullSecretRef:
+    name: pull-secret <11>
+----
+<1> Specify the name for the `ClusterDeployment` resource.
+<2> Specify the base domain that the cluster should belong to.
+<3> Specify the name of the `ImageClusterInstall` in which you defined the container images to use for the image-based installation of the target host.
+<4> What does this do???
+<5> Specify the `Secret` resource containing the admin kubeconfig for this cluster.
+<6> Specify the `Secret` resource containing the admin kubeconfig for this cluster.
+<7> Specify the cluster ID for this cluster.
+<8> What's this?
+<9> Specify a name for the cluster.
+<10> Specify the deployment platform for the cluster.
+<11> Specify the name of the pull secret to use when pulling container images.
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-deployment.yaml
+----
+
+. Create the `ManagedCluster` resource:
+
+.. Create a YAML file that defines the `ManagedCluster` resource:
++
+.Example `ibi-managed.yaml` file
+[source,yaml]
+----
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: sno-ibu-00 <1>
+spec:
+  hubAcceptsClient: true <2>
+----
+<1> Specify the name for the `ManagedCluster` resource.
+<2> Specify `true` to enable {rh-rhacm} to mange the cluster.
+
+.. Create the `ManagedCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-managed.yaml
+----
+
+.Verification
+
+. TODO. Get status in hub cluster, SSH to node and watch reconfiguration? What steps to do these?
```

Comment:
You can watch the conditions on the `ImageClusterInstall` to determine the progress of the installation.
To SSH you'd need to have provided that public key in the `ImageClusterInstall` spec.